pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/question/DatePicklistDef.java

```diff
@@ -75,12 +88,18 @@ public int getStartYear() {
     }
 
     /**
-     * Get end year (years forward from year anchor).
+     * Get end year (years forward from year anchor), truncated to current year if future years are not allowed.
```

Comment:
I added the `allow_future_years` flag so we can configure date question picklist to start at 2019 and not exceed current year. This flag is not visible to clients -- it's only used on server-side to compute the year range for date questions.